src/main/java/games/strategy/triplea/ui/screen/UnitsDrawer.java

```diff
@@ -130,6 +130,8 @@ public void draw(final Rectangle bounds, final GameData data, final Graphics2D g
                 null);
           }
           break;
+        default:
+          throw new AssertionError("unknown unit flag draw mode: " + drawUnitNationMode);
```

Comment:
In this case, it is used to indicate unreachable code.  Or stated another way, if somehow you do reach this code, make sure we blow things up as much as possible so someone notices.  We regularly catch `Exception`, so throwing an unchecked exception might go unnoticed.  We should never be catching `Throwable` (although we do in about eight locations), so an `Error` will, at a minimum, kill the current thread, which should cause someone to notice. :smile:

I originally observed this pattern with respect to enums in Item 30 of _Effective Java_.  However, the Guava team has a [page](https://github.com/google/guava/wiki/ConditionalFailuresExplained) on recommendations for which types of exceptions to throw in various circumstances that is more complete, IMO.  The usage in this PR pretty much falls into the "Impossible condition" category.